starter-code/public/javascripts/map.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+function map() {
+  console.log("AIzaSyAvUL7rvwQhc3aXnVdRbXrEBTWGQhkeHII");
+}
+
+function startMap() {
+  var ironhackBCN = {
+    lat: 41.3977381,
+    lng: 2.190471916
+  };
+  var map = new google.maps.Map(
+    document.getElementById('map'), {
+      zoom: 15,
+      center: ironhackBCN
+    }
+  );
+
+}
+
+
+var myMarker = new google.maps.Marker({
```

Comment:
Maybe you want to do this inside startMap(). That way, this code is not executed until the page loads and you are sure that the map exists. Furthermore, you avoid making myMarker a global variable.